packages/flutter/lib/src/material/bottom_sheet.dart

```diff
@@ -18,17 +18,28 @@ const double _kCloseProgressThreshold = 0.5;
 const Color _kTransparent = const Color(0x00000000);
 const Color _kBarrierColor = Colors.black54;
 
-class _BottomSheetDragController extends StatelessComponent {
-  _BottomSheetDragController({
+class BottomSheet extends StatelessComponent {
+  BottomSheet({
     Key key,
-    this.performance,
-    this.child,
-    this.childHeight
-  }) : super(key: key);
+    this.performance, // will be mutated
```

Comment:
Given our history with mismanaging Performance ownership, I'm all in favour of more documentation. I added some more detail as Adam suggested.